pages/main.py

```diff
@@ -216,6 +218,9 @@ def render_main():
 
 @app.route('/QA')
 def render_QA_page():
+    # Check if the connection to the neemhub is established
```

Comment:
I think this needs a bigger comment as it is not obvious how a function called "connect_mongo" is used for "checking" the connection

Reply:
so you could add that this will cause an exception, and how is this then handled

Reply:
I added a bigger comment